src/lightning_app/utilities/network.py

```diff
@@ -48,11 +48,12 @@ def _configure_session() -> Session:
     return http
 
 
-def _check_service_url_is_ready(url: str, timeout: float = 1) -> bool:
+def _check_service_url_is_ready(url: str, timeout: float = 5) -> bool:
     try:
         response = requests.get(url, timeout=timeout)
         return response.status_code in (200, 404)
     except (ConnectionError, ConnectTimeout, ReadTimeout):
+        print("Not ready")
```

Comment:
What is the intention here? Would you like to give feedback to the user? Or perhaps a debug logging message is more appropriate?